common-tasks/usb.md

```diff
@@ -383,6 +293,185 @@ When you finish, detach the device:
 
 This feature is not yet available in Qubes Manager however, if you would like to contribute to Qubes OS project by implementing it and are a student please consider applying for the [Google Summer of Code][gsoc-page] scholarship and choosing QubesOS Project as a mentor organization. You can find list of our our Project Ideas [here][project-page].
 
+Creating and Using a USB qube
+-----------------------------
+
+**Warning:** This has the potential to prevent you from connecting a keyboard to Qubes via USB. There are problems with doing this in an encrypted install (LUKS). If you find yourself in this situation, see this [issue][2270-comm23].
+
+Connecting an untrusted USB device to dom0 is a security risk since dom0,
+like almost every OS, reads partition tables automatically. The whole
+USB stack is put to work to parse the data presented by the USB device in order
+to determine if it is a USB mass storage device, to read its configuration, etc.
+This happens even if the drive is then assigned and mounted in another qube.
+
+To avoid this risk, it is possible to prepare and utilize a USB qube.
+
+A USB qube acts as a secure handler for potentially malicious USB devices,
+preventing them from coming into contact with dom0 (which could otherwise be
+fatal to the security of the whole system). With a USB qube, every time you
+connect an untrusted USB drive to a USB port managed by that USB controller, you
+will have to attach it to the qube in which you wish to use it (if different
+from the USB qube itself), either by using Qubes VM Manager or the command line
+(see instructions above). 
+You can create a USB qube using the management stack by performing the following
+steps as root in dom0:
+
+ 1. Enable `sys-usb`:
+
+        sudo qubesctl top.enable qvm.sys-usb
+
+ 2. Apply the configuration:
+
+        sudo qubesctl state.highstate
```

Comment:
Better compress this to one line action: `sudo qubesctl state.sls qvm.sys-usb`
This will also not leave `qvm.sys-usb` enabled and applied on every next `state.highstate` call.